.github/ISSUE_TEMPLATE.md

```diff
@@ -1,11 +1,22 @@
-### What did you expect to happen?
+<!-- Please use this template when creating an issue. 
+- Please check the boxes after you've created your issue.
+- Please use the latest version of Yoast SEO.-->
```

Comment:
Because we don't want too many checkboxes and the latest version of Yoast SEO is already mentioned in the guidelines and in the issue at the bottom itself.